README.rst

```diff
@@ -21,6 +21,22 @@
 
 Get coala to lint all your languages in your project with one tool and config!
 
+Introduction
+------------
+
+coala is a language independent analysis toolkit. It empowers developers
+to create rules which a project's code should conform to. coala takes care
+of showing these issues to users in a friendly manner, is versatile and
+can be used in any environment. Patches to automatically fix code will
+be managed too. coala has a set of official bears (plugins) to provide
+an out-of-the-box analysis functionality for many popular languages in
+addition to some generically applicable algorithms.
+
+For information on the languages supported by coala-bears, refer to this link
```

Comment:
i think the hyperlink should be on ``this link`` and not on ``here``, but that's just an opinion. the ``here`` word seems too much here